src/lib/iterm.c

```diff
@@ -26,9 +26,12 @@ int iterm_rebuild(sprixel* s, int ycell, int xcell, uint8_t* auxvec){
 }
 
 // spit out the control sequence and data.
-int iterm_draw(const ncpile *p, sprixel* s, FILE* out, int y, int x){
-  if(goto_location(p->nc, out, y, x)){
-    return -1;
+int iterm_draw(const tinfo* ti, const ncpile *p, sprixel* s, FILE* out, int y, int x){
+  (void)ti;
```

Comment:
this ought fix bitmap placement on iTerm, which was apparently broken. @michaelsbradleyjr does that track with what you've seen? before this change, they always ought have been emitted at the current cursor location.

Reply:
nevermind, it already had the `goto_location()` and thus probably already worked.

Reply:
In my experience it was already working.